lib/page/performMouseAction.js

```diff
@@ -30,6 +30,20 @@ const performMouseAction = async function performMouseAction(context, actionType
     return ({ status: STATUS.INVISIBLE });
   }
 
+  const overlappingNode = await context.page.evaluate(selectorNode => {
+    selectorNode.scrollIntoView({ block: 'center' });
+
+    const { left, top } = selectorNode.getBoundingClientRect();
+    const x = left + (selectorNode.clientWidth / 2);
+    const y = top + (selectorNode.clientHeight / 2);
+
+    const nodeEl = document.elementFromPoint(x, y);
+
+    return nodeEl && !selectorNode.contains(nodeEl) && nodeEl.className;
+  }, visibleElements[0]);
+
+  if (overlappingNode) return ({ status: `${selector} is overlapped by ${overlappingNode}` });
```

Comment:
`classname` may contain spaces, so the status will be like:

```
.my-button is overlapped by btn-link tabnav-tab write-tab js-write-tab px-3 px-sm-6 px-md-3 flex-1 flex-md-auto
```

This is probably not the status we want to get.

Reply:
Now it will display only first className.

Reply:
Wouldn't it better to join the classnames with a dot?

```
.my-button is overlapped by .btn-link.tabnav-tab.write-tab.js-write-tab.px-3.px-sm-6.px-md-3.flex-1.flex-md-auto
```

In this case it's easier to look up for the node in the DOM 🤔

Reply:
Fixed.